ui/app/src/views/admin/AdminView.tsx

```diff
@@ -21,11 +21,11 @@ function AdminView() {
   const { tab } = useParams();
   return (
     <Stack sx={{ width: '100%' }} m={2} gap={2}>
-      <AppBreadcrumbs admin={true} />
+      <AppBreadcrumbs rootPageName="Admin" />
```

Comment:
```suggestion
      <AppBreadcrumbs rootPageName="Administration" />
```
Let's use the same name as the title, no?